src/dndbeyond/content-scripts/character.js

```diff
@@ -43,9 +43,20 @@ function rollAbilityOrSavingThrow(paneClass, rollType) {
     const ability_name = ability_string.split(" ")[0];
     const ability = ability_abbreviations[ability_name];
     let modifier = $("." + paneClass + "__modifier .ct-signed-number,." + paneClass + "__modifier .ddbc-signed-number").text();
+    let remarkable_athlete_used = false;
+
+    if (rollType == "ability" &&
+        character.hasClassFeature("Remarkable Athlete") &&
+        character.getSetting("champion-remarkable-athlete", false) &&
+        (ability == "STR" || ability == "DEX" || ability == "CON")) {
+        const remarkable_athlete_mod = Math.ceil(character._proficiency / 2);
+        modifier += " + " + remarkable_athlete_mod;
+        remarkable_athlete_used = true;
+    }
```

Comment:
I would just put the JoaT in the `else` with a comment that they don't stack, instead of having a `remarkable_athlete_used` flag.